frontend/src/pages/storageClasses/StorageClassesPage.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+import {
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateBody,
+  PageSection,
+  Title,
+  Alert,
+  AlertActionCloseButton,
+} from '@patternfly/react-core';
+
+import { Annotation, StorageClassConfig } from '~/k8sTypes';
+import useStorageClasses from '~/concepts/k8s/useStorageClasses';
+import { ProjectObjectType, typedEmptyImage } from '~/concepts/design/utils';
+import ApplicationsPage from '~/pages/ApplicationsPage';
+import { updateStorageClassConfig } from '~/services/StorageClassService';
+
+const StorageClassesPage: React.FC = () => {
+  const [storageClasses, storageClassesLoaded, storageClassesError] = useStorageClasses();
+  const [isAlertOpen, setIsAlertOpen] = React.useState(false);
+
+  const defaultStorageClass = storageClasses.find(
+    (storageClass) =>
+      storageClass.metadata.annotations?.[Annotation.StorageClassIsDefault] === 'true',
+  );
+
+  // Open default class alert when no default class exists
+  React.useEffect(() => {
+    setIsAlertOpen(!defaultStorageClass?.metadata.name);
+  }, [defaultStorageClass?.metadata.name]);
+
+  // Add storage class config annotations automatically for all storage classes without them
+  React.useEffect(() => {
+    storageClasses.forEach((storageClass, index) => {
+      const { metadata } = storageClass;
+      const { name: storageClassName } = metadata;
+
+      if (!metadata.annotations?.[Annotation.OdhStorageClassConfig]) {
+        const isDefault = defaultStorageClass?.metadata.uid === metadata.uid;
+        const storageClassConfig: StorageClassConfig = {
+          displayName: storageClassName,
+          ...(!defaultStorageClass
```

Comment:
imagine someone adds a new storage class after we have already configured the configs for the previous classes. 

if there was no openshift default set, then the previous should all be set to enabled. which is what you have here

but now you later add a new storage class, what happens, becuase there is still no default, it will set it as enabled, however you would have already st a ODH default class. 

so what i am coming to is that we should only enable if there is no odh default or there is not openshift default

wdyt

Reply:
That makes sense. Will update.